pkg/flink/handler.go

```diff
@@ -149,6 +144,15 @@ func flinkClusterTaskInfo(ctx context.Context, flinkCluster *flinkOp.FlinkCluste
 		taskLogs = append(taskLogs, tl...)
 	}
 
+	if jmi := flinkCluster.Status.Components.JobManagerIngress; jmi != nil {
+		customInfoMap["jobmanager-urls"] = jmi.URLs
```

Comment:
where's `jobmanager-urls` coming from?

Reply:
this is a new property that we are exposing to `flyte` with the ingress generated backend URLs for Jobmanager UI/API.

`flyteconsole` can't display these values atm but they are available through `flyteadmin` API.